inject.js

```diff
@@ -10,21 +10,21 @@ function waitForApp() {
 
             // Get the stats
             const knownStatsArray = staticData.knownStats || [];
-            const knownStats = knownStatsArray.find(stat => stat.id === "explicit");
-
-            if (!knownStats) {
-                console.error("Error: Could not find 'explicit' stats in knownStats.");
-                return;
-            }
-
-            // Build the stats map
-            const statsMap = knownStats.entries.reduce((map, stat) => {
-                if (map[stat.text]) {
-                    map[stat.text].push(stat.id);
-                } else {
-                    map[stat.text] = [stat.id];
-                }
 
+            // Build the stats map from 'explicit' and 'desecrated' knownStats
+            const statsMap = knownStatsArray.reduce((map, knownStat) => {
+                if (!knownStat.entries) return map;
+                if (knownStat.id !== 'explicit' && knownStat.id !== 'desecrated') return map;
```

Comment:
Actually I think there's a true assumption that an explicit mod variant exists for all desecrated mods. I will remove this.